src/typedefs.php

```diff
@@ -11,16 +11,6 @@
 
 namespace HHVM\UserDocumentation;
 
-use type Facebook\DefinitionFinder\ScannedDefinition;
-
-type PHPDotNetAPIIndexEntry = shape(
```

Comment:
not HHAPIDoc, but yes, dead - search used to intentionally include links to php.net

Reply:
rest is mostly hhapidoc